third_party/cf-cli-glob/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/google/kf/third_party/cf-cli-glob
```

Comment:
I think this change should have caused VENDOR-LICENSE to be updated, can you check that?